ReleaseTooling/README.md

```diff
@@ -30,21 +30,22 @@ for that version. Check with `xcodebuild -version`.
 
 ### Launch Arguments
 
-See `main.swift` and the `LaunchArgs` struct for information on specific launch arguments.
+See `main.swift`  for information on specific launch arguments,  or use  `swift run zip-builder --help`.
 
-You can pass in launch arguments with Xcode by clicking "zip-builder" beside the Run/Stop buttons, clicking "Edit
-Scheme" and adding them in the "Arguments Passed On Launch" section.
+You can pass in launch arguments with Xcode by clicking "zip-builder" beside the Run/Stop buttons, clicking
+"Edit Scheme" and adding them in the "Arguments Passed On Launch" section.
 
 #### Common Arguments
 
 These arguments assume you're running the command from the `ReleaseTooling` directory.
 
 **Required** arguments:
-- `-templateDir $(pwd)/Template`
-  - This should always be the same.
+
+- `--repo-dir <PATH_TO_firebase_ios_sdk_REPO>`
```

Comment:
This shouldn't be required in the --zip-pods use case

Reply:
I believe we still need the `Template` directory though, for compilation purposes. Is that incorrect? I agree it's not ideal and I'd love to change it later but I think that's currently the case. I'll have to check though...

Reply:
Yep, agreed. Disregard my comment.

Reply:
Ah I think I know what happened. Previously we had to specify both the root of the repo and the template, but I made the template path auto-generated from the root of the repo. I can revert that in this PR or a follow up, your call. Looks like when using the `zip-pods` passing in the template is easier (and used for building Analytics)

Reply:
`--repo-dir` makes specifying the template shorter anyway so I'm fine to always specify it and update analytics instructions.

Ideally the template would be built-in, but at least when I thought about it a while ago, it was non-trivial.

Reply:
Sounds good, will leave it as is, update the internal docs and evaluate separating or building in the template at a later date. Thanks!